openbas-front/src/admin/components/simulations/simulation/injects/InjectIndex.tsx

```diff
@@ -96,12 +97,24 @@ const InjectIndexComponent: FunctionComponent<{ exercise: ExerciseType; injectRe
               label={t('Execution details')}
               className={classes.item}
             />
+            {
+              injectResultOverviewOutput.inject_type !== 'openbas_email' && injectResultOverviewOutput.inject_type !== 'openbas_ovh_sms' && injectResultOverviewOutput.inject_type !== 'openbas_mastodon' && injectResultOverviewOutput.inject_type !== 'openbas_http_query' && (
```

Comment:
```suggestion
              !['openbas_email', 'openbas_ovh_sms', 'openbas_mastodon', 'openbas_http_query'].includes(injectResultOverviewOutput.inject_type)  && (
```
I did not test but wouldn't it be more readable this way ?